chisel/src/executor.rs

```diff
@@ -357,6 +384,44 @@ fn format_token(token: Token) -> String {
     }
 }
 
+/// Formats a [pt::EventDefinition] into an inspection message
+///
+/// ### Takes
+///
+/// An borrowed [pt::EventDefinition]
+///
+/// ### Returns
+///
+/// A formatted [pt::EventDefinition] for use in inspection output.
+///
+/// TODO: Verbosity option
+fn format_event_definition(event_definition: &pt::EventDefinition) -> Result<String> {
+    let event_name = event_definition.name.as_ref().expect("Event has a name").to_string();
+    let inputs = event_definition
+        .fields
+        .iter()
+        .map(|param| {
+            let name = param
+                .name
+                .as_ref()
+                .map(ToString::to_string)
+                .unwrap_or_else(|| "<anonymous>".to_string());
+            let kind = Type::from_expression(&param.ty)
+                .and_then(Type::into_builtin)
+                .ok_or_else(|| eyre::eyre!("Invalid type in event {event_name}"))?;
+            Ok(ethabi::EventParam { name, kind, indexed: param.indexed })
+        })
+        .collect::<Result<Vec<_>>>()?;
+    let event = ethabi::Event { name: event_name, inputs, anonymous: event_definition.anonymous };
+
+    Ok(format!(
+        "Type: {}\n├ Name: {}\n└ Signature: {:?}",
+        Paint::red("event"),
+        Paint::cyan(&event.name),
+        event.signature()
```

Comment:
Could we also color this cyan to keep consistency among the styling of the output?

![Screenshot from 2023-03-16 11-12-57](https://user-images.githubusercontent.com/8406232/225662015-88583842-d638-4bc8-b45f-3c70590317dc.png)

Reply:
I believe the color you are referring to is set elsewhere in the codebase in existing code not touched by this PR. The color from the code snippet above (` Paint::red("event")`) is for the `Type: event` output, which matches the red color of `Type: bytes32`.